src/gui/userstatus.cpp

```diff
@@ -28,12 +28,31 @@ namespace OCC {
 
 UserStatus::UserStatus(QObject *parent)
     : QObject(parent)
-    , _status("online")
+    , _status(Status::Online)
     , _message("")
 {
 
 }
 
+QString UserStatus::enumToString(const Status status) const
+{
+    return QVariant::fromValue(status).toString();
+}
+
+UserStatus::Status UserStatus::stringToEnum(const QString &statusText) const 
```

Comment:
@camilasan Just wondering, wouldn't it be easier to just use `QVector` to map `enum` values to `QString`?

Reply:
I just changed it, please check it again.